php-checks/src/main/java/org/sonar/php/checks/formatting/ControlStructureSpacingCheck.java

```diff
@@ -75,15 +74,15 @@ private void checkSpaceForStatement(FormattingStandardCheck formattingCheck, Ast
     for (AstNode semicolon : node.getChildren(PHPPunctuator.SEMICOLON)) {
       Token semicolonToken = semicolon.getToken();
       Token nextToken = semicolon.getNextAstNode().getToken();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "nextToken" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "nextToken". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1854)